Bottom: 1aef012868c491d970c4e5b6c1bce85ba64ab235
Top:    d1c89b8880e66b5118d53362958d27bc8b0f0601
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 13:04:15 +0100

Temporary debug for toporouter - switch back to cmil Coord units


---

diff --git a/src/const.h b/src/const.h
index c82e90a..f554fd4 100644
--- a/src/const.h
+++ b/src/const.h
@@ -68,10 +68,10 @@
 #define LN_2_OVER_2		0.346573590
 
 /* PCB/physical unit conversions */
-#define COORD_TO_MIL(n)	((n) / 25400.0)
-#define MIL_TO_COORD(n)	((n) * 25400.0)
-#define COORD_TO_MM(n)	((n) / 1000000.0)
-#define MM_TO_COORD(n)	((n) * 1000000.0)
+#define COORD_TO_MIL(n)	((n) / 100.0)
+#define MIL_TO_COORD(n)	((n) * 100.0)
+#define COORD_TO_MM(n)	((n) * 0.000254)
+#define MM_TO_COORD(n)	((n) / 0.000254)
 #define COORD_TO_INCH(n)	(COORD_TO_MIL(n) / 1000.0)
 #define INCH_TO_COORD(n)	(MIL_TO_COORD(n) * 1000.0)
 
diff --git a/src/toporouter.c b/src/toporouter.c
index b732791..38135a6 100644
--- a/src/toporouter.c
+++ b/src/toporouter.c
@@ -232,7 +232,7 @@ toporouter_arc_class(void)
   return klass;
 }
 
-#define MARGIN 10.0f
+#define MARGIN 10.0f /* XXX: Not sure - seems to be used to offset cairo drawing - thus could be in Coords */
 
 drawing_context_t *
 toporouter_output_init(int w, int h, char *filename)
